contracts/src/fp/OPSuccinctFaultDisputeGame.sol

```diff
@@ -510,4 +510,9 @@ contract OPSuccinctFaultDisputeGame is Clone, ISemver {
     function genesisL2BlockNumber() external view returns (uint256 genesisL2BlockNumber_) {
         genesisL2BlockNumber_ = GENESIS_L2_BLOCK_NUMBER;
     }
+
+    /// @notice Returns the proof reward.
+    function proofReward() external view returns (uint256 proofReward_) {
+        proofReward_ = PROOF_REWARD;
+    }
```

Comment:
What is the purpose of these getters (and the getters above).

Does `IDisputeGame` require them?

If not, you should just be able to read the `proofReward` by calling `PROOF_REWARD`.

Reply:
Need getters for implementing challenger to challenge with correct bond amount. The immutable variables are `internal`, so I added the getters.